docs/beyond_basics/101-147-riastores.rst

```diff
@@ -0,0 +1,704 @@
+
+   In order to define an alias for an individual dataset in a store, one needs
+   to create an ``alias/`` directory in the root of the datastore and place
+   a :term:`symlink` of the desired name to the dataset inside of it. Here is how it is
+   done, for the midterm project dataset:
+
+   First, create an ``alias/`` directory in the store:
+
+   .. runrecord:: _examples/DL-101-147-121
+      :language: console
+      :workdir: dl-101
+
+      $ mkdir /home/me/myriastore/alias
+
+   Afterwards, place a :term:`symlink` with a name of your choice to the dataset
+   inside of it. Here, we create a symlink called ``midterm_project``:
+
+   .. runrecord:: _examples/DL-101-147-122
+      :language: console
+      :workdir: dl-101
+      :realcommand: echo "$ ln -s /home/me/myriastore/$(datalad -C DataLad-101/midterm_project -f'{infos[dataset][id]}' wtf | sed 's/^\(...\)\(.*\)/\1\/\2/') /home/me/myriastore/alias/midterm_project" && ln -s /home/me/myriastore/$(datalad -C DataLad-101/midterm_project -f'{infos[dataset][id]}' wtf | sed 's/^\(...\)\(.*\)/\1\/\2/') /home/me/myriastore/alias/midterm_project
+
+   Here is how it looks like inside of this directory:
+
+   .. runrecord:: _examples/DL-101-147-123
+      :language: console
+      :workdir: dl-101
+
+      $ tree /home/me/myriastore/alias
+
+   Afterwards, the alias name, prefixed with a ``~``, can be used as a dataset
+   identifier:
+
+   .. runrecord:: _examples/DL-101-147-124
+      :language: console
+      :workdir: dl-101
+
+      datalad clone ria+file:///home/me/myriastore#~midterm_project
+
+   This makes it easier for others to clone the dataset and will provide a sensible
+   default name for the clone if no additional path is provided in the command.
+
+The dataset clone is just like any other dataset clone. Contents stored in
+:term:`Git` are present right after cloning, while the contents of annexed files
+is not yet retrieved from the store and can be obtained with a :command:`datalad get`.
+
+.. runrecord:: _examples/DL-101-147-125
+   :language: console
+   :workdir: dl-101
+
+   $ cd myclone
+   $ tree
+
+To demonstrate file retrieval from the store, let's get an annexed file:
+
+.. runrecord:: _examples/DL-101-147-126
+   :language: console
+   :workdir: dl-101/myclone
+
+   $ datalad get books/progit.pdf
+
+
+.. findoutmore:: What about creating RIA stores and cloning from RIA stores with different protocols
+
+   Consider setting up and populating a RIA store on a server via the ``file``
+   protocol, but cloning a dataset from that store to a local computer via
+   ``SSH`` protocol. Will this be a problem for file content retrieval?
+   No, in all standard situations, DataLad will adapt to this. Upon cloning
+   the dataset with a different protocol than it was created under,
+   enabling the special remote will initially fail, but DataLad will adaptively
+   try out other protocols to enable the ora-remote and retrieve file contents.
```

Comment:
"try out other protocols"

Technically that's not correct. What `datalad clone` does, is to try to enable the special remote using the same URL it cloned from. That may or may not include a protocol change. Could also be a change in hostname or path.

Reply:
thx